shared/message/platform/darwin/DarwinSensors.proto

```diff
@@ -128,22 +186,51 @@ message DarwinSensors {
         Servo headPan        = 19;
         Servo headTilt       = 20;
     }
-
-    google.protobuf.Timestamp timestamp       = 1;
-    uint32                    cm740ErrorFlags = 2;
-    LEDPanel                  ledPanel        = 3;
-    HeadLED                   headLED         = 4;
-    EyeLED                    eyeLED          = 5;
-    Buttons                   buttons         = 6;
-    float                     voltage         = 7;
-    Accelerometer             accelerometer   = 8;
-    Gyroscope                 gyroscope       = 9;
-    FSRs                      fsr             = 10;
-    Servos                    servo           = 11;
+    /// Timestamp when our data was taken
+    google.protobuf.Timestamp timestamp = 1;
+    /// Holds the error code (if any) from the CM740
+    /// The list of errors is here: https://emanual.robotis.com/docs/en/dxl/protocol1/#status-packetreturn-packet
+    uint32 cm740ErrorFlags = 2;
+    /// Toggle status of the LEDs on the panel
+    LEDPanel ledPanel = 3;
+    /// Toggle status of the HeadLED
+    HeadLED headLED = 4;
+    /// Toggle status of the EyeLED
+    EyeLED eyeLED = 5;
+    /// Push-down status of the left and middle buttons
+    Buttons buttons = 6;
+    /// Voltage for the whole robot, from the main battery
+    float voltage = 7;
+    /// The value is between 0 and 1023, representing a value between -4g and 4g => 512 = 0g.
+    /// Is {x, y, z} acceleration
+    Accelerometer accelerometer = 8;
+    /// The value is between 0 and 1023, represent a value between -500 degrees per second and 500 degrees per second.
+    /// => 512 = 0 degrees per second. Is {x, y, z} angular velocity
+    Gyroscope gyroscope = 9;
+    /// Pair of sets of FSR sensors for both feet, to detect if they are down
+    FSRs fsr = 10;
+    /// Set of the robot's servos
+    Servos servo = 11;
 }
 
 // Button press events
+
+/**
+ * Empty message to indicate button press
+ */
```

Comment:
Would be good to get some info on what up/down mean. From what I understand, up is when it's been pressed, and down is when it's just stopped being pressed?